### DISCUSSION TITLE
Domain name is not working even though it is setup in DigitalOcean

### DISCUSSION THREAD

**Discussion by cptalpdeniz** [getredash/redash#6632] [Q&A] [answered]
I did not have this issue when I installed the official image. Using the local build, even though I have the nameservers and A records etc. set up, it is not forwarding to correct location. My .env file has the following entries too.
```
REDASH_HOST=https://chartedbi.com
REDASH_SERVER_NAME=https://chartedbi.com
```

The first tab is when it's accessed by the IP.
![image](https://github.com/getredash/redash/assets/17345222/f9431c41-10b4-4a06-bf2e-10edc3be8bbe)

![image](https://github.com/getredash/redash/assets/17345222/b819b194-d571-40dd-a5ad-e618ff6e7612)

![image](https://github.com/getredash/redash/assets/17345222/a590228a-0d9a-4607-a277-a5695bd7995b)

**Comment by masayuki038** [comment]
Hi @cptalpdeniz 
Please let us know the URL of the local build page you referred to. And please confirm your server listen 443 port.

**Reply by masayuki038** [reply]
I noticed you referred to https://github.com/getredash/redash/wiki/Local-development-setup in https://github.com/getredash/redash/discussions/6625.

This is the document for setting up local development environment. Therefore, it doesn't support TLS. For TSL, please check this page.

- https://redash.io/help/open-source/admin-guide/https-ssl-setup
- https://gist.github.com/arikfr/64c9ff8d2f2b703d4e44fe9e45a7730e

**Reply by cptalpdeniz** [reply]
I will check it, however the problem is even if the DNS is set up (just like before when I installed Redash using the official image) I cannot access it via the domain name. I can only access it using the `IP:5000`.

**Reply by guidopetri** [reply]
That's correct, you have to port forward or use a reverse proxy like nginx

**Reply by cptalpdeniz** [reply]
Okay I will try that and reply with the result here. One thing that is confusing is that there is a `docker-compose.yml` file in /opt/redash but also in the root of the source code.

**Reply by cptalpdeniz** [reply]
While following the GIST, after step 8, I got this error
```
Failed authorization procedure. chartedbi.com (http-01): urn:ietf:params:acme:error:connection :: The server could not
connect to the client to verify the domain :: XXX.XXX.XXX.XXX: Fetching http://chartedbi.com/.well-known/acme-challenge/nrs1SZea9bxFXNkTPykjtN90c6p2rVqzWWu2XD6CblY: Timeout during connect (likely firewall problem)
```

I have changed the IP for security purposes.

**Comment by guidopetri** [comment]
I would strongly, strongly recommend reading more about what ports are and how port forwarding works, as well as the basics of LE/getting HTTPS working. It feels like the issue here is with networking, and we can't help you with that.

**Reply by cptalpdeniz** [reply]
Indeed it was due to networking, the `docker-compose.yml` file did not have nginx entries. Check my comment below.

**Comment by cptalpdeniz** [comment, marked as answer]
UPDATE: I realized the `docker-compose.yml` file in the root of the source code did not have nginx image section, added that and voila! it worked.
